libs/langchain/langchain/chat_models/bedrock_anthropic.py

```diff
@@ -0,0 +1,31 @@
+from typing import Any, Dict, List, Optional
+
+from langchain.callbacks.manager import CallbackManagerForLLMRun
+from langchain.chat_models.anthropic import ChatAnthropicMessageConverter
+from langchain.chat_models.bedrock import BaseBedrockChat
+from langchain.schema.messages import AIMessage, BaseMessage
+from langchain.schema.output import ChatGeneration, ChatResult
+
+
+class BedrockChatAnthropic(BaseBedrockChat):
+    @property
+    def _llm_type(self) -> str:
+        """Return type of chat model."""
+        return "bedrock-anthropic-chat"
```

Comment:
If we are using the string `amazon_bedrock` for our normal LLM class, shouldn't we use the same convention of using underscore as the separator and naming so like `amazon_bedrock_anthropic_chat`?

Reply:
I agree with being consistent for the amazon bedrock naming and will include this.

Note: There a decent amount of llm and chat classes that have "_llm_type" properties with dashes in their name. Should these be consistent across all llm and chat classes?

Reply:
I think so. Why does Bedrock have underscores?

Reply:
Not sure... It might not have been intentional.